src/util/FormField.js

```diff
@@ -180,15 +180,15 @@ const FormMultilineTextField = (props) => (
 const onDateChange = (prevDate, fieldName, setFieldValue) => (event) => {
     const newValue = event.target.value;
     const date = prevDate ? prevDate : new Date();
-    const time = format(date).format("HH:mm:ss");
+    const time = formatDateObject(date, "YYYY-MM-DD");
```

Comment:
This changes the format of this `time` string from `hours:minutes:seconds` to `year-month-day`.

Notice below in the `setFieldValue` call that `fieldName` is getting set with the string `${newValue} ${time}`, which should be in the format `YYYY-MM-DD HH:mm:ss`.